### PR TITLE
netdata: add format

### PR DESCRIPTION
**Pull request by lasers**
This adds format: string to print (default '[\?color=stuffs goes here]')

TODO: Add a missing separator. What magic to use? Pls.
TODO: Fix docstring (obv).
TODO: New PR for module enhancements after this...
└──────── Add new threshold options and such.

Ongoing efforts to FORMAT ALL THE THINGS! (meme). #98

### CODE REVIEW

File: py3status/modules/netdata.py [hunk 1]
```diff
@@ -3,32 +3,38 @@
...
-    Get system status.
+    """Get system status.
     """
     def __init__(self, nic):
         self.nic = nic
 
     def netBytes(self):
-        """
-        Get bytes directly from /proc.
+        """Get bytes directly from /proc.
```
Comment (tobes) [inline py3status/modules/netdata.py]: This is ugly

File: py3status/modules/netdata.py [hunk 2]
```diff
@@ -43,19 +49,36 @@ class Py3status:
...
-    low_traffic = 400
-    med_speed = 60
-    med_traffic = 700
+    format = u'[\?color=down LAN(Kb): {down}↓ {up}↑] ' \
+             u'[\?color=total T(Mb): {download}↓ {upload}↑ {total}↕]'
     nic = None
+    thresholds = {
+        'down': [(0, "bad"), (30, "degraded"), (60, "good")],
+        'total': [(0, "good"), (400, "degraded"), (700, "bad")]
+    }
```
Comment (tobes) [inline py3status/modules/netdata.py]: this needs creating from the old config params if they are supplied, but get things happily working first

File: py3status/modules/netdata.py [hunk 3]
```diff
@@ -3,32 +3,38 @@
...
-    nic: the network interface to monitor (default None)
-
-Color options:
-    color_bad: Rate is below low threshold
-    color_degraded: Rate is below med threshold
-    color_good: Rate is med threshold or higher
+    cache_timeout: refresh interval for this module (default 2)
+    format: display format for this module
+        *(default '[\?color=down LAN(Kb): {down}↓ {up}↑] '
+        '[\?color=total T(Mb): {download}↓ {upload}↑ {total}↕]'*
```
Comment (tobes) [inline py3status/modules/netdata.py]: missing the closing `)`

File: py3status/modules/netdata.py [hunk 4]
```diff
@@ -3,32 +3,38 @@
...
+    nic: network interface to use (default None)
+    thresholds: color thresholds to use
+        *(default '{down': [(0, "bad"), (30, "degraded"), (60, "good")], '
```
Comment (tobes) [inline py3status/modules/netdata.py]: `'` int the wrong place `{'down':` the whole thing is a dict not a string - you have extra `'`s knocking about

**Comment by tobes** [comment]
this should fix the tests till we get a better docstring parser

```
diff --git a/py3status/modules/netdata.py b/py3status/modules/netdata.py
index 440aa2d..5c82c0b 100644
--- a/py3status/modules/netdata.py
+++ b/py3status/modules/netdata.py
@@ -5,7 +5,8 @@ Display network speed and bandwidth usage.
 Configuration parameters:
     cache_timeout: refresh interval for this module (default 2)
     format: display format for this module
-        (default '[\?color=down LAN(Kb): {down}↓ {up}↑] [\?color=total T(Mb): {download}↓ {upload}↑ {total}↕])'
+        *(default '[\?color=down LAN(Kb): {down}↓ {up}↑]
+         [\?color=total T(Mb): {download}↓ {upload}↑ {total}↕]')*
     nic: network interface to use (default None)
     thresholds: color thresholds to use
         *(default {'down': [(0, 'bad'), (30, 'degraded'), (60, 'good')],
@@ -50,7 +51,8 @@ class Py3status:
     """
     # available configuration parameters
     cache_timeout = 2
-    format = u'[\?color=down LAN(Kb): {down}↓ {up}↑] [\?color=total T(Mb): {download}↓ {upload}↑ {total}↕]'
+    format = u'[\?color=down LAN(Kb): {down}↓ {up}↑] ' + \
+        u'[\?color=total T(Mb): {download}↓ {upload}↑ {total}↕]'
     nic = None
     thresholds = {
         'down': [(0, 'bad'), (30, 'degraded'), (60, 'good')],
diff --git a/tests/test_module_doc.py b/tests/test_module_doc.py
index b6c776e..1891f7d 100644
--- a/tests/test_module_doc.py
+++ b/tests/test_module_doc.py
@@ -24,6 +24,7 @@ IGNORE_MODULE = [
 # Ignored items will not have their default values checked or be included for
 # alphabetical order purposes
 IGNORE_ITEM = [
+    ('netdata', 'format'),  # line too long for docstring parsing
     ('screenshot', 'save_path'),  # home dir issue
     ('rate_counter', 'config_file'),  # home dir issue
     ('group', 'format'),  # dynamic depending on click_mode
```

**Comment by ultrabug** [comment]
Alright, let's move forward thanks !

**Pull request merged**